exporter/signalfxexporter/config_test.go

```diff
@@ -92,6 +95,29 @@ func TestLoadConfig(t *testing.T) {
 			},
 		},
 		DeltaTranslationTTL: 3600,
+		Correlation: &correlation.Config{
+			HTTPClientSettings: confighttp.HTTPClientSettings{
+				Endpoint: "",
+				Timeout:  5 * time.Second,
+			},
+			AccessToken:         "",
```

Comment:
Do we need a separate access token here? Can we use the top-level one?

Reply:
If it's not set we default it to the top-level value. (This isn't documented in the README.md which I can add). The reason I didn't delete access token in correlation config is we use the config struct to pass to configure the correlation tracker so I would have to add another struct to capture the access token. This also keeps symmetry with endpoint where endpoint can be configured in the correlation settings but defaults to the top-level one.

I don't feel that strongly either way so if you think it's better to just have the top-level one I can remove it.

Reply:
Up to you, but it feels superfluous (unless there is a real use case where there is a need to use a different access token for correlation requests).